cmd/cmd.go

```diff
@@ -47,6 +47,7 @@ func NewCertManagerCtlCommand(ctx context.Context, in io.Reader, out, err io.Wri
 			return logf.ValidateAndApply(logOptions)
 		},
 		SilenceErrors: true, // Errors are already logged when calling cmd.Execute()
+		SilenceUsage:  true, // Don't print usage when an error occurs
```

Comment:
Instead of calling `os.Exit(1)` when the command fails, we now actually properly shutdown.
This means that cobra now has the time to print the Usage on each error. To prevent this, we disable this option here.